data/campaigns/Descent_Into_Darkness/achievements.cfg

```diff
@@ -1,12 +1,59 @@
 #textdomain wesnoth-did
+
+# achievements in DiD generally should be based around exploration and trying funny things in the campaign
+# these achievements shouldn't be so much about battle challenges, I want to give players another incentive to explore the little details
+# also the achievements are somewhat written as if from Malin's point of view which is why there's some weird and sometimes quirky flavor in them
```

Comment:
Some of this should be a `# po:` hint, otherwise the translators won't see it. Maybe put in to the first achievement.

Reply:
Okay done.